airflow/dags/utils/error_handling.py

```diff
@@ -17,33 +18,48 @@ def handle_dag_failure(context: Dict) -> None:
     `context` is a dictionary of the kind returned by `get_template_context`. For details see:
     https://github.com/databricks/incubator-airflow/blob/master/airflow/models.py
     """
-    post_alert_to_discord(context)
+    dag_id = context["task_instance"].dag_id
+    task_id = context["task_instance"].task_id
+    log_url = context["task_instance"].log_url
 
+    logging.info(
+        f"Handling DAG failure: dag_id: {dag_id},"
+        f"task_id: {task_id}, log_url: {log_url}, context: {context}",
+    )
+
+    platform = Variable.get(f"alert_platform", "discord")
```

Comment:
```suggestion
    platform = Variable.get("alert_platform", "discord")
```

Reply:
Thanks! Fixed as per suggestion. 

I think test fix is going to be a separate task. From the recently closed PRs, we are ignoring the failed test. @gulshngill